en/_posts/2016-04-20-quick-start-with-marathon.md

```diff
@@ -6,16 +6,16 @@ author: devatoria
 lang: en
 ---
 
-Welcome to the quick start tutorial of the PaaS Docker lab. This document will guide you to launch your first application and play with it.
+Welcome to the quick start tutorial of the PaaS Docker lab. This document will help you launch your first application using the Marathon Web UI.
 
-#Access to your interface
+#Access to your Web UI
```

Comment:
suggestion: make the "Marathon" explicit. That will improve readability for people "scanning though" your tutorial. For example "Access your Marathon Web UI"